### PR TITLE
Feature/main feature

### PR DESCRIPTION
**Pull request by pacyL2K19**
# The aim of this PR
- Build the `HTML` through  js 
- Consume the API to fetch weather info 
- Setup the linter 
- Update minimal style to the main boxes

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+homePage();
+
+const defaultCountry = 'Moscow';
+const getWeather = (query) => {
+  const alertBox = document.getElementById('alert');
+
+  fetch(`${weatherApi.base}weather?q=${query}&units=metric&APPID=${weatherApi.key}`)
+    .then(weather => weather.json()).then(displayWeather)
+    .catch(() => {
+    //   console.log('here');
```
Comment (afanimartin) [inline src/index.js]: - Remove this comment and any other. Comments make the codebase to look ugly.

File: src/scripts/render.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+  colLeftWrapper.appendChild(colLeft);
+
+  const colRightWrapper = document.createElement('div');
+  colRightWrapper.className = 'details col-sm-4 mt-3 py-3 d-flex align-items-center';
+
+  /**
+   * @card that displays details
+   */
+
```
Comment (afanimartin) [inline src/scripts/render.js]: - Remove this comment

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Amazing work on this PR. Your app looks good.
### Well done 🌵 🌵 🌵 
- Codebase is modularised
- Use of ES6 syntax
- A user can search weather for a particular city
- There's a toggle button for changing temperature units

### Changes to make 🙅🏿 🙅🏿 🙅🏿 
- You have comments in your codebase. Comments make the code look ugly.
- Add error handling for the API request.

### Nice to have
- Move the temperature display below the date display. I couldn't see the result and was about to ask for it as a change to be made.


### Martin Afani
#### Slack: whiz25

**Review by voscarmv — APPROVED** [review]
# Project status: Approved 💯 ✨ 😎 👍 
### You may merge this branch and move forward ⛙🛣️ 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻
# Happy Coding! 🧠 ⌨️ 🖱️ 🖥️

**Review by pacyL2K19 — COMMENTED** [review]
# [Personal] Additional :heavy_plus_sign: 
- Avoid ES5 symtaxes as much as possible

**Pull request merged**